src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -75,6 +79,45 @@ class MinikubeImpl implements Minikube {
     status(): Promise<MinikubeInfo> {
         return minikubeStatus(this.context);
     }
+
+    checkUpgradeAvailable() {
+        return minikubeUpgradeAvailable(this.context);
+    }
+}
+
+async function minikubeUpgradeAvailable(context: Context) {
+    if (!await checkPresent(context, CheckPresentMode.Alert)) {
+        // not installed, no upgrade.
+        return;
+    }
+
+    const sr = await context.shell.exec(`${context.binPath} update-check`);
+    if (sr.code != 0) {
```

Comment:
Think you need triple-equals to satisfy the linter.

```suggestion
    if (sr.code !== 0) {
```